system/web/Renderer.cfc

```diff
@@ -694,6 +694,29 @@ component accessors="true" serializable="false" extends="coldbox.system.Framewor
 		return locateView(arguments.view);
 	}
 
+	public string function elixir( required string fileName, string buildDirectory = 'build' ) {
+		var includesConvention = controller.getSetting( "includesConvention", true );
```

Comment:
So...something like this?
```cfc
var mapping = event.getCurrentModule() != "" ? event.getModuleRoot() : controller.getSetting( "appMapping" );

var href = "#mapping#/#includesConvention#/#arguments.path#";
```
Or does the `appMapping` need to be there regardless, more like:
```cfc
var appMapping = controller.getSetting( "appMapping" );
var moduleMapping = event.getCurrentModule() != "" ? event.getModuleRoot() & "/" : "";

var href = "#appMapping#/#moduleMapping##includesConvention#/#arguments.path#";
```